Foundation/NSCFDictionary.swift

```diff
@@ -139,8 +143,12 @@ internal func _CFSwiftDictionaryContainsValue(dictionary: AnyObject, value: AnyO
 internal func _CFSwiftDictionaryGetKeysAndValues(dictionary: AnyObject, keybuf: UnsafeMutablePointer<Unmanaged<AnyObject>?>, valuebuf: UnsafeMutablePointer<Unmanaged<AnyObject>?>) {
     var idx = 0
```

Comment:
not that it is likely but if both the keybuf and valuebuf are nil you could early return before iterating